framework/src/modules/chain/bft/bft.js

```diff
@@ -124,17 +124,8 @@ class BFT extends EventEmitter {
 			this.finalityManager.maxHeight - this.finalityManager.minHeight <
 			this.constants.activeDelegates * BFT_ROUND_THRESHOLD
 		) {
-			const tillHeight = this.finalityManager.minHeight - 1;
-			const fromHeight =
-				this.finalityManager.maxHeight -
-				// Search is inclusive, therefore, it should start from one above (ex: 288 - 500, which results total 303)
-				this.constants.activeDelegates * BFT_ROUND_THRESHOLD +
-				1;
-			await this._loadBlocksFromStorage({
```

Comment:
Why we need to keep this function still int he file? actually that function should have been renamed to `fillCache`

Reply:
@ManuGowda Also mentioned this, but this logic is slightly different than fill cache because it fills to the top, and this is patch so i didn't want to clean up. This will be changed in my BFT PR #4737